src/stories/06-save-buttons.storie.tsx

```diff
@@ -14,3 +14,9 @@ export const clickableSaveButton = () =>
     onClick={()=>{window.alert("Save")}}
   />;
 
+export const labeledSaveButton = () =>
```

Comment:
Unrelated to this PR, but shouldn't the name of this file be *.stories.tsx?

Reply:
Ah, yes!  Thanks!

Reply:
Fixed.